pkg/cmd/backupdriver/cli/install/install.go

```diff
@@ -71,6 +75,9 @@ func NewInstallOptions() *InstallOptions {
 		PodCPULimit:    pkgInstall.DefaultBackupDriverPodCPULimit,
 		PodMemLimit:    pkgInstall.DefaultBackupDriverPodMemLimit,
 		PVSecret:       false,
+		MasterAffinity: false,
+		HostNetwork:    false,
+		Features:       "",
```

Comment:
We may not need to explicitly initialize them here since variables are implicitly initialized to their corresponding zero values in Go.